rfcs/XXXX-gene-sets/text.md

```diff
@@ -0,0 +1,100 @@
+- LogFold - A ratio of the difference in expression levels. Typically calculated by taking the log2 of the gene expression value for each cell and then finding the mean of that value for each set of cells. Because the data is already log2 transformed you can then take the difference between the two means as the logfold value 
+## Problem Statement | Background
+
+Gene sets are the expected output of differential expression analysis in cellxgene. Two (non-overlaping) sets of cells 
+are selected and the difference in the expression level for each gene is computed by comparing the average 
+expression level for each set of cells. When there is a particularly large difference in expression between the two sets of 
+cells, that gene is identified and presented to the user. To ensure consistency and compatibility it is necessary to define a 
+data format for storing references to these genes and any accompanying statistical data. 
+
+## [Optional] Product Requirements
+### Users should be able to export differential expression results
+Currently cellxgene users can choose two sets of cells and compute/display the log fold and adjusted p-value of the top ten differentially expressed genes. Additionally we display a histogram of the gene expression levels of the selected cells for each gene in the right side bar. 
+- [US1] A user should be able to export the list of 10 genes along with their logfold and adjusted p-values in a csv file (see data formats below for more detail) by clicking a button
+### Users should be able to upload their own gene sets and see them in the right side bar with the option to apply them within current cellxgene functionalities, if the same gene ontology was used when generating the current dataset and the geneset uploaded by the user (ie the gene names match) 
+- [US2] A user should be able to upload a geneset via a csv file (see data formats below for more detail) for a dataset.
+- [US2.5] A user should be able to upload a geneset they previously downloaded from cellxgene
+- [UC3] When a user uploads a geneset, that list of genes should appear in the right side bar 
+- [US4] If the user was logged in when they uploaded a geneset that genesit should persist if they logout and back in or reload the page
+- [US5] If the gene names match those used in the dataset the user should be able to use general cellxgene functionality (color by genes, calculate differential expression levels, etc.) on those genes
+- [US6] Genesets uploaded for one dataset should be available in other datasets?
+
+### If there is a standard format for differential expression output, we should follow that. If not, the format should be as straightforward as possible while serving these above cases.
+
+### [Optional] Nonfunctional requirements
+- Users should be able to export precomputed differential expression results in real time
+- Users should be able to upload gene sets and have them immediately appear in the right side bar
```

Comment:
I think this entire section can be deleted since it's not related to the data model.